benchmark/matmul.cpp

```diff
@@ -252,11 +292,13 @@ static void NvFuserScheduler_Matmul_4warp3stage(
 
 static void NvFuserScheduler_Matmul_8warp3stage(
     benchmark::State& benchmark_state,
-    MatmulLayout layout) {
+    MatmulLayout layout,
+    int splitk_factor = 1) {
   auto cta_tile = GemmTile(256, 128, 32);
   int number_of_stage = 3;
```

Comment:
Where there is no

```C++
  if (splitk_factor == -1) {
    int M = benchmark_state.range(0);
    int N = benchmark_state.range(1);
    splitk_factor = computeAutoSplitKFactor(M, N, cta_tile.m, cta_tile.n);
  }
```

Reply:
Thanks. I had only run for 4warp3stage. I'll add the other three. Actually I think we can just take warps and stages as parameters for this function. I'll try and reorganize these benchmarks.